webpack.config.js

```diff
@@ -31,10 +31,11 @@ if (process.env.NODE_ENV === 'production') {
 
 module.exports = {
     entry: {
-        base: "./src/base-deps.js",
-        codemirror: "./src/codemirror-deps.js",
-        igv: "./src/igv-deps.js",
-        markdown: "./src/markdown-deps.js"
+        base: "./src/js/base-deps.js",
+        codemirror: "./src/js/codemirror-deps.js",
+        igv: "./src/js/igv-deps.js",
+        markdown: "./src/js/markdown-deps.js",
+        pipeline: "./src/js/pipeline-deps.js"
```

Comment:
Seems like a +1 for me, but not an expert/not going to veto over this

Reply:
I'm going to leave this in place, since I don't know when I'll next get a chance to put in a PR against UI, and I think this will make things more straightforward for whoever comes after me.